src/Checkbox.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import cx from 'classnames';
+import Flex from './Flex';
+import FlexItem from './FlexItem';
+
+/**
+ * @module Checkbox
+ */
+class Checkbox extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			checked: props.checked || false
+		};
+		this.onChange = this.onChange.bind(this);
+	}
+
+	onChange(e) {
+		this.setState({ checked: e.target.checked });
+	}
+
+	render() {
+		const {
+			checked, // eslint-disable-line no-unused-vars
+			children,
+			className,
+			id,
+			label,
+			name,
+			value,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			'minTouchHeight',
+			className
+		);
+
+		const elId = id || `${name}-${value}`;
+
+		return (
+			<Flex align='center'
+				className={classNames}
+				{...other}>
+				<FlexItem shrink>
+					<input	type='checkbox'
+						name={name}
+						value={value}
+						checked={this.state.checked}
+						id={elId}
+						onChange={this.onChange}
+					/>
+				</FlexItem>
+				<FlexItem>
+					<label className='label--minor' htmlFor={elId}>{label}</label>
```

Comment:
Go ahead and use `label--minor` in this component; I'll make sure the class will make it into `swarm-sasstools` with this PR: https://github.com/meetup/meetup-web-components/pull/80

This has been filed in Jira as: https://meetup.atlassian.net/browse/SDS-157

Once `swarm-sasstools` is merged into this repo, the class will be defined and your labels will get prettier.